src/notification/NotificationController.ts

```diff
@@ -0,0 +1,86 @@
+import BaseController, { BaseConfig, BaseState } from '../BaseController';
+
+interface viewedNotification {
+[id: number]: boolean;
```

Comment:
The indentation is off throughout this file :thinking: It's missing here, and it's four spaces rather than two in most other places.

I'm surprised this wasn't caught by the linter

Reply:
Fixed the indentation problem in this file. I have been using the `tab`  key previously.